### PR TITLE
feat: add `meta` support for healthcheck instance method `add_target`

### PR DESCRIPTION
**Pull request by leslie-tsang**

### CODE REVIEW

File: lib/resty/healthcheck.lua [hunk 1]
```diff
@@ -366,8 +366,9 @@ end
 -- default is `true`.
 -- @param hostheader (optional) a value to use for the Host header on
 -- active healthchecks.
+-- @param tbl_meta (optional) a lua table with custom info of business stuff
 -- @return `true` on success, or `nil + error` on failure.
-function checker:add_target(ip, port, hostname, is_healthy, hostheader)
+function checker:add_target(ip, port, hostname, is_healthy, hostheader, tbl_meta)
```
Comment (starsz) [inline lib/resty/healthcheck.lua:371]: Can we use `meta` rather than `tbl_meta`?

Comment (leslie-tsang) [reply lib/resty/healthcheck.lua:371]: LGTM, `tbl_meta` means `table` type

Comment (leslie-tsang) [reply lib/resty/healthcheck.lua:371]: IMO, The `tbl_` prefix helps others to better understand what they are doing when looking at the code or making modifications.

Comment (starsz) [reply lib/resty/healthcheck.lua:371]: OK.

**Review by starsz — APPROVED** [review]

**Pull request merged**